virttest/utils_test/libvirt.py

```diff
@@ -3647,16 +3647,23 @@ def check_logfile(search_str, log_file, str_in_log=True,
                                           log_file))
 
 
-def check_qemu_cmd_line(content, err_ignore=False):
+def check_qemu_cmd_line(content, err_ignore=False,
+                        cmd_params=None, runner_on_target=None):
     """
     Check the specified content in the qemu command line
     :param content: the desired string to search
     :param err_ignore: True to return False when fail
                        False to raise exception when fail
+    :param cmd_parms: The parms for remote executing
+    :param runner_on_target:  Remote runner
     :return: True if exist, False otherwise
     """
     cmd = 'pgrep -a qemu'
-    qemu_line = process.run(cmd, shell=True).stdout_text
+    if not(cmd_params and runner_on_target):
```

Comment:
runner_on_target is an optional argument for remote.run_remote_cmd(), if there is no runner_on_target, it will create a runner from cmd_params. So how about checking it only from cmd_params?
BTW, I think it'll be great if you can change the param for remote to remote_args or remote_dict, like that.

Reply:
Changed it to remote_params. But I think it's better to use runner_on_target, or it's a little mixed up to use the params.

Reply:
Hmm, I feel a little wired because the runner_on_target is an optional argument in remote.run_remote_cmd(), but for now, remote_params and runner are mandatory here. @dzhengfy  would you mind sharing your thoughs?

Reply:
Thanks for your comments, @Yingshun . Check the remote.run_remote_cmd() again and the function can be run with one of them exists, only None must be set for params.  Change it to (remote_params or runner_on_target), how do you think of it?